config_defaults.yml

```diff
@@ -29,13 +29,25 @@
     env: PUPPET_GEM_VERSION="~> 3.0"
     bundler_args: --without system_tests
 Gemfile:
+  supports_windows:
+    - puppetlabs-acl
+    - puppetlabs-chocolatey
+    - puppetlabs-dism
+    - puppetlabs-dsc
+    - puppetlabs-powershell
+    - puppetlabs-reboot
+    - puppetlabs-registry
+    - puppetlabs-sqlserver
+    - puppetlabs-wsus_client
```

Comment:
I would prefer `supports_windows` to go away completely, but this is also in scope for MODULES-4027 (create a set of gems encapsulating our ruby-version-specific dependencies), so I'm supportive of adding a ugly hack here to get this merged.